app/controllers/reservations_controller.rb

```diff
@@ -78,7 +78,12 @@ def index
     if session[:all_dates]
       time = source
     else
-      time = source.starts_on_days(@start_date, @end_date)
+
+      # error occurs here:
```

Comment:
I'm assuming these comments are only intended for review? You don't necessarily need to put them in the code since the GitHub diff view will allow us to easily identify what went out and what went in, but I like the thought. We should remove them in the finalized diff.